extensions/geode-modules-session-internal/src/main/java/org/apache/geode/modules/session/internal/jmx/SessionStatistics.java

```diff
@@ -44,14 +44,6 @@ public long getRegionUpdates() {
     return regionUpdates.get();
   }
 
-  public void setActiveSessions(int sessions) {
```

Comment:
You mentioned earlier that if a user were very sly that they could write their own JMX component to read our stats.  Would this change be of concern in that case?  I'm not sure we should even really code around that theoretical corner case anyways.

Reply:
I don't think we should concern about this one: the actual interface `SessionStatisticsMXBean` (what a user should actually use) never had the method in the first place and, moreover, it does't make much sense to provide public JMX methods to set the actual number of total and active sessions anyway.